javascripts/govuk/show-hide-content.js

```diff
@@ -0,0 +1,150 @@
+(function (global) {
+  'use strict';
+
+  var $ = global.jQuery;
+  var GOVUK = global.GOVUK || {};
+
+  function ShowHideContent() {
+    var self = this;
+
+    // Radio and Checkbox selectors
+    var selectors = {
+      namespace: 'ShowHideContent',
+      radio: '.block-label input[type="radio"]',
+      checkbox: '.block-label input[type="checkbox"]'
```

Comment:
Hi @colinrotherham, thanks for taking the time to add this JS and the tests and documentation to accompany it. We'll happily take it from here, rather than pester you with change requests. Thanks for your help!